### PR TITLE
Add is_incident_report flag

### PR DESCRIPTION
**Pull request by cesarvarela**
Adds a new field `is_incident_report`  to reports. which should be `true` for reports linked to an incident and `false`  otherwise.

### CODE REVIEW

File: site/gatsby-site/migrations/2022.09.19T20.06.43.is-incident.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ *
+ * @param {{context: {client: import('mongodb').MongoClient}}} context
+ */
+
+exports.up = async ({ context: { client } }) => {
+  await client.connect();
+
+  const reportsCollection = client.db(config.realm.production_db.db_name).collection('reports');
+
+  await reportsCollection.updateMany({}, { $set: { is_incident_report: true } });
```
Comment (pdcp1) [inline site/gatsby-site/migrations/2022.09.19T20.06.43.is-incident.js:13]: We assume all existing reports are linked to an incident. Is that correct?

File: site/gatsby-site/migrations/2022.09.19T20.06.43.is-incident.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const config = require('../config');
+
+/**
+ *
+ * @param {{context: {client: import('mongodb').MongoClient}}} context
+ */
+
+exports.up = async ({ context: { client } }) => {
```
Comment (pdcp1) [inline site/gatsby-site/migrations/2022.09.19T20.06.43.is-incident.js:8]: Does it make sense to have an `export.down` in this case?

**Review by pdcp1 — CHANGES_REQUESTED** [review]

**Comment by pdcp1** [comment]
@cesarvarela the integrity test is failing, please review.

image

File: site/gatsby-site/migrations/2022.09.19T20.06.43.is-incident.js (hunk 1 above)
Comment (cesarvarela) [reply site/gatsby-site/migrations/2022.09.19T20.06.43.is-incident.js:13]: yes

**Comment by cesarvarela** [comment]
@pdcp1 yes the test will fail until the migration is ran, which only happens once merged to staging

File: site/gatsby-site/migrations/2022.09.19T20.06.43.is-incident.js (hunk 2 above)
Comment (cesarvarela) [reply site/gatsby-site/migrations/2022.09.19T20.06.43.is-incident.js:8]: I think it does not because it is not used by anything anyway

Comment (pdcp1) [reply site/gatsby-site/migrations/2022.09.19T20.06.43.is-incident.js:8]: OK, that makes sense to me too.

**Review by pdcp1 — APPROVED** [review]

**Pull request closed without merging**